src/sidebar/components/ShareDialog/ExportAnnotations.tsx

```diff
@@ -133,17 +163,45 @@ function ExportAnnotations({
           >
             Name of export file:
           </label>
-          <Input
-            data-testid="export-filename"
-            id={fileInputId}
-            defaultValue={defaultFilename}
-            value={customFilename}
-            onChange={e =>
-              setCustomFilename((e.target as HTMLInputElement).value)
-            }
-            required
-            maxLength={250}
-          />
+          <div className="flex">
+            <Input
+              classes={classnames('grow', { 'mr-2': exportFormatsEnabled })}
+              data-testid="export-filename"
+              id={fileInputId}
+              defaultValue={defaultFilename}
+              value={customFilename}
+              onChange={e =>
+                setCustomFilename((e.target as HTMLInputElement).value)
+              }
+              required
+              maxLength={250}
+            />
+            {exportFormatsEnabled && (
+              <div className="grow-0">
+                <SelectNext
+                  value={selectedExportFormat}
+                  onChange={setSelectedExportFormat}
+                  buttonContent={selectedExportFormat.name}
+                  classes={classnames(
+                    // Setting a fixed width so that the size does not change when
```

Comment:
Could this end up cutting off the text if the font is larger than expected? Perhaps better to set min-width instead.

Reply:
It could potentially cut the text, but then it would render the text ellipsis.

I set a size which fits the currently longer option, because setting min width does not seem to properly work when inside a flex container (I'm trying to verify that's the reason though). It could also be the class is not being set in the right place, in which case https://github.com/hypothesis/frontend-shared/pull/1369 might help.

Reply:
Turns out `min-w-20` was not working because tailwind doesn't define that 🤦🏼‍♂️ https://tailwindcss.com/docs/min-width

I have defined `min-w-[5rem]` instead, as I have seen we use this pattern frequently. 5 rem is the same size set by `w-20`.